packages/beacon-state-transition/src/block/processAttestationsAltair.ts

```diff
@@ -93,6 +93,17 @@ export function processAttestationsAltair(
       if (totalWeight > 0) {
         totalBalanceIncrementsWithWeight += effectiveBalanceIncrements[index] * totalWeight;
       }
+
+      if ((flagsNewSet & TIMELY_TARGET) === TIMELY_TARGET) {
+        const validator = state.validators.get(index);
+        if (inCurrentEpoch) {
+          if (!validator.slashed && isActiveValidator(validator, currentEpoch)) {
```

Comment:
Is the `isActiveValidator` check necessary here?

Reply:
removed 👍